.travis.yml

```diff
@@ -0,0 +1,48 @@
+dist: bionic
+language: python
+jdk: openjdk11
+cache:
+  - pip
+  - directories:
+    - $HOME/.m2
+env:
+  global:
+  - AWS_REGION="us-east-1"
+  - AWS_DEFAULT_REGION=$AWS_REGION
+
+before_install:
+  - >
+    pip install --upgrade pip 'attrs==19.2.0'
+    -r https://raw.githubusercontent.com/aws-cloudformation/cloudformation-cli/master/requirements.txt
+install:
+  - pip install .
+
+script:
+  - pre-commit run --all-files
+
+jobs:
+  include:
+    - python: "3.6"
+      language: python
+    - python: "3.7"
+      language: python
+    - python: "3.8"
+      language: python
+    - language: java
+      before_install: skip
+      install: skip
+      script: mvn verify
+    - stage: integ
```

Comment:
doesn't this step need a python version specified? getting:

>  This job ran using the default Python version, which was updated to 3.6 on April 16th, 2019. You can explicitly stay on the previous version by specifying python: 2.7 in your .travis.yml.

might be good to pin it

Reply:
good idea will do